apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/block-links-map.ts

```diff
@@ -0,0 +1,255 @@
+	},
+
+	'a8c/blog-posts': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/blog-posts-block/',
+	},
+
+	'jetpack/send-a-message': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/whatsapp-button-block/',
+	},
+
+	/**
+	 * Jetpack Blocks
+	 */
+	'jetpack/timeline': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/timeline-block/',
+	},
+
+	'jetpack/story': { url: 'https://wordpress.com/support/wordpress-editor/blocks/story-block/' },
+
+	'jetpack/revue': { url: 'https://wordpress.com/support/wordpress-editor/blocks/revue-block/' },
+
+	'jetpack/rating-star': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/ratings-block/',
+	},
+
+	'jetpack/related-posts': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/related-posts-block/',
+	},
+
+	'jetpack/repeat-visitor': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/repeat-visitor-block/',
+	},
+
+	'jetpack/podcast-player': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/podcast-player-block/',
+	},
+
+	'jetpack/opentable': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/opentable-block/',
+	},
+
+	'jetpack/map': { url: 'https://wordpress.com/support/wordpress-editor/blocks/map-block/' },
+
+	'jetpack/image-compare': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/image-compare-block/',
+	},
+
+	'jetpack/gif': { url: 'https://wordpress.com/support/wordpress-editor/blocks/gif-block/' },
+
+	'jetpack/event-countdown': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/event-countdown-block/',
+	},
+
+	'jetpack/donations': { url: 'https://wordpress.com/support/wordpress-editor/blocks/donations/' },
+
+	'jetpack/calendly': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/calendly-block/',
+	},
+
+	'jetpack/business-hours': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/business-hours-block/',
+	},
+
+	'jetpack/wordads': { url: 'https://wordpress.com/support/wordpress-editor/blocks/ad-block/' },
+
+	'jetpack/payments-intro': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/payments/',
+	},
+
+	'jetpack/contact-info': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/contact-info-block/',
+	},
+
+	'jetpack/tiled-gallery': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/tiled-gallery-block/',
+	},
+
+	'jetpack/slideshow': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/slideshow-block/',
+	},
+
+	'jetpack/subscriptions': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/subscription-form-block/',
+	},
+
+	'jetpack/contact-form': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/form-block/',
+	},
+
+	'jetpack/layout-grid': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/layout-grid-block/',
+	},
+
+	'jetpack/mailchimp': {
+		url: 'https://wordpress.com/support/wordpress-editor/blocks/mailchimp-block/',
+	},
```

Comment:
May be useful to run https://marketplace.visualstudio.com/items?itemName=richie5um2.vscode-sort-json against this object.

Reply:
I could, but I thought it would be better to organize them by source ( core, WordPress.com, Jetpack and other ). The reason being is that there are a few that don't follow the same naming convention. But if you think it would be better I can sort this.